src/main/java/codeu/model/data/User.java

```diff
@@ -106,4 +112,34 @@ public String getTime() {
     return date;
   }
 
+  /** When a User is initialized, this method creates a private Conversation with every other User that already exists. */
+  public void createConversations() {
+    // get all USers from UserStore
+    // for each User, create a new Conversation with this User and set to private
+    // add all Conversations to ConversationStore
+    // writethrough to DataStore
+    UserStore userStore = UserStore.getInstance();
+    List<User> users = userStore.getUsers();
+    ConversationStore conversationStore = ConversationStore.getInstance();
+    List<Conversation> conversations = conversationStore.getAllConversations();
+
+    for (User u: users) {
+      String otherUserName = u.getName();
+      String firstUser;
```

Comment:
you could shorten this a bit by initing these variables with one of the 2 cases, then have a single if case to swap them as needed. Best to avoid declaring local variables without initialising them whenever possible

Reply:
I see, that makes sense and looks more concise too, thank you @tucosh ! Will fix that :)